storage/ops.go

```diff
@@ -192,6 +207,7 @@ func (bd *BlockDevice) WritePartitionTable(legacyBios bool) error {
 		"optimal",
 		bd.GetDeviceFile(),
 		"--script",
+		"--",
```

Comment:
What is this change fixing?

Reply:
The "--" is required to tell parted that there are no more options, else the use of '-1' for the script is treated as a command line option.